### PR TITLE
Move from travis to github actions

### PR DESCRIPTION
**Pull request by mvanlonden**
https://github.com/graphql-python/graphene/issues/1127

Use github action for linting, deploying and testing. Removes travis.

Fixes automated deploys.

### CODE REVIEW

**Comment by mvanlonden** [comment]
@jetzhou to unblock v3 migration

File: tox.ini [hunk 1]
```diff
@@ -1,20 +1,40 @@
 [tox]
-envlist = pre-commit,py{27,35,36,37}-sql{11,12,13}
+envlist = pre-commit,py{27,35,36,37,38,39}-sql{12,13},flake8
 skipsdist = true
 minversion = 3.7.0
 
+[gh-actions]
+python =
+    2.7: py27
+    3.5: py35
```
Comment (jkimbo) [inline tox.ini:9]: These can probably be removed since they're not being tested against.

**Review by jkimbo — APPROVED** [review]
LGTM. The code coverage doesn't seem to be working though.

**Comment by richin13** [comment]
Can we get this merged? 🙏🏼 I'd like to run the tests on PR #317 (currently they're passing locally). This is important to get graphene to v3

File: tox.ini (hunk 1 above)
Comment (mvanlonden) [reply tox.ini:9]: Will follow up with coverage fix and removing these tests

**Pull request merged**